### ISSUE TITLE
🆕 Software Suggestion | Pi-hole configuration

### ISSUE THREAD

**Issue by kaushalyap** [privacytoolsIO/privacytools.io#1811]
## Basic Information

Name: Pi-hole
Category : Software
URL : https://pi-hole.net/

## Description

- https://github.com/pi-hole/pi-hole
- https://docs.pi-hole.net/

## Why I am making the suggestion

- Protect things at network level rather than at the browser level.
- [Can use as DNS server](https://docs.pi-hole.net/main/post-install), instead of depending on third parties.
- Have [dashboard to with records](https://github.com/pi-hole/AdminLTE).

## My connection with the software

No connection, but seems most people on reddit use it so community may help with docs.

## Suggested Website Content

- Software listing
- How to configure pi-hole on suggested Linux distros (Fedora, Debian)

**Comment by beerisgood** [comment]
Unbound should be listed in combination with PiHole as with Unbound no external DNS providers are needed, which is important for privacy

**Comment by kaushalyap** [comment]
@beerisgood What is unbound, link please!

**Comment by beerisgood** [comment]
> @beerisgood What is unbound, link please!

Realy? You link to PiHole but don't know this? The PiHole team itself build a how to about.

Also first result in search engine..
https://docs.pi-hole.net/guides/unbound/

**Comment by kaushalyap** [comment]
> Realy? You link to PiHole but don't know this? The PiHole team itself build a how to about.
Nope, I thought unbound a separate product. Anyway thanks for sharing the link.
